app/javascript/App/components/NewsfeedPage/CoinListItem.js

```diff
@@ -20,11 +28,12 @@ export default (props) => {
         <div className="b f5 pl2">{coin.get('symbol')}</div>
         {coin.get('market_info') && (
           <div className="right-align">
-            <Currency>{coin.getIn(['market_info', 'price_usd'])}</Currency>
-            <PercentageChange
-              value={coin.getIn(['market_info', 'percent_change_24h'])}
+              {coinPrice && <div>$<span>{coinPriceFixed}</span></div>}
+            {!coinPrice && <div style={{fontSize:'.75rem'}}>UNLISTED</div>}
```

Comment:
👾 Hm formatting looks weird...

Also, can we just use the `.smaller3` CSS class instead of hardcoding 0.75rem?

Finally, the height of the Watchlist items that are `UNLISTED` are a bit shorter than those with prices, could you make them the same height?

Reply:
👍